.github/workflows/ci.yml

```diff
@@ -17,7 +17,7 @@ name: CI
 on: [push, pull_request]
 
 env:
-  GO_PACKAGES: ./services/... ./indexer/... ./bitcoin/... ./configuration/...
+  GO_PACKAGES: ./services/... ./indexer/... ./bitcoin/... ./dogecoin/...
```

Comment:
`configuration` changed to `dogecoin` instead of `bitcoin` changing to `dogecoin`, is this intended?

Reply:
This same thing applies to `Makefile`.